packages/notifications/src/error.test.ts

```diff
@@ -0,0 +1,121 @@
+import { IgnoreErrorDecider, makeNotificationsContract } from './error';
+import { Notifier } from './useNotifications';
+
+const mockNotifications: Notifier = {
+  notify: jest.fn(),
+};
+
+const ignoreError: IgnoreErrorDecider = (title: string, _obj?: unknown): boolean => {
+  return title === 'Ignore Me';
+};
+describe('reportError', () => {
+  it('calls notify and console.error', async () => {
+    // Arrange
+    jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
I err on the side of caution with this... since putting it in beforeEach would silence ALL tests console errors, which can be bad... so intentionally putting it in each test instead.

Reply:
Fair enough. Though I'd claim that since this is the `reportError` describe block, I'll assume that every test calls `reportError` and we'll always want to silence console errors for tests in this block.